supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
+* Users first: Allows users to navigate intuitively.
+* Allows Google to understand architecture of the site.
+* Establishes a quantifiable pathway to recognize topical expertise.
+* They increase site visitation time.
+* Helps Google crawl your site.
+* Builds relevancy and ranking signals for keywords and topics.
+* Builds citation or helps users add context.
+
+.Technical best practices
+* Use standard hyperlinks:
+`<a href="http://www.same-domain.com/" title="Keyword Text">Keyword Text</a>`
+* Do not use non hyperlinks.
+* Do not use JavaScript links. 
+* Do not use Flash Links.
+* Do not use links in Frames or iframes
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Search engines have a soft-cap of crawling around 150 links. Avoid over-linking and consider that the navigational links are crawlable too. 
+
+.Anchor text best practices
```

Comment:
What do you mean by "anchor text"? Is this the text of the hyperlink (e.g. `<a href="url">Anchor text</a>`?)

Reply:
Yep, the link text is what is meant by anchor text. Again though, it's not my content so that's an educated guess!

Reply:
Yeah I think we might need a  better word rather than "Anchor text". I didn't understand what this meant the first time I read it either. Because when I think of anchor, I think of the #<value> after the URL that takes you down to the anchor for a particular section, but saying anchor text makes me think you're still talking about that actual anchor.

I looked in the IBM style guide for how they refer to it and they say about using a "meaningful descriptive label for the link", and also the "link text".

So I'd probably go with "Link text" instead of "Anchor text".